### PR TITLE
Scale BH spectra by the `bolometric_luminosity`

### PR DESCRIPTION
**Pull request by stephenmwilkins**
At present BH spectra - except in UnifiedAGN - are unscaled (scaled by unity) since previously they were the true luminosities predicted by the model. However, this was changed in favour of having grids normalised to unity. Consequently this PR now scales all BH by the `bolometric_luminosity`. 

This requires a bit more testing to check for impacts on UnifiedAGN and would be cleaner if previous PRs (which are merged here) are merged to main first.

## Issue Type
- [x] Bug

## Checklist
- [x] I have read the [CONTRIBUTING.md]() -->
- [x] I have added docstrings to all methods
- [x] I have added sufficient comments to all lines
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no pep8 errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/synthesizer/sed.py [hunk 1]
```diff
@@ -1314,18 +1314,25 @@ def calculate_ionising_photon_production_rate(
         x = self._lam
```
Comment (WillJRoper) [inline src/synthesizer/sed.py:1314]: Can't merge this because it includes #761

File: src/synthesizer/particle/blackholes.py [hunk 2]
```diff
@@ -396,9 +396,9 @@ def _prepare_sed_args(
             for prop in props
         ]
 
-        # For black holes mass is a grid parameter but we still need to
-        # multiply by mass in the extensions so just multiply by 1
-        mass = np.ones(npart, dtype=np.float64)
+        # For black holes the grid Sed are normalised to 1.0 so we need to
+        # scale by the bolometric luminosity.
+        bol_lum = self.bolometric_luminosity.value
```
Comment (WillJRoper) [inline src/synthesizer/particle/blackholes.py:401]: You also need to do this in `parametric/blackhole.py` in the same functions at lines 198 and 335.

**Review by WillJRoper — CHANGES_REQUESTED** [review]
Looks great. Just needs the changes mirrored in the parametric object and #761 to be removed. Just run:
```
git checkout main -- src/synthesizer/sed.py
```
to remove those changes.

File: src/synthesizer/particle/blackholes.py (hunk 2 above)
Comment (stephenmwilkins) [reply src/synthesizer/particle/blackholes.py:401]: Done

**Comment by WillJRoper** [comment]
This is ready but can't be merged because it still contains #761 (including the updates). You can just checkout the normal version of those files to remove these changes.

**Review by WillJRoper — APPROVED** [review]

**Pull request merged**